models/account.py

```diff
@@ -55,3 +53,19 @@ def IdSet(self, value):
     def id(self):
         if self.key is not None:
             return self.key.string_id()
+
+
+    # fields below are not part of MVP and maybe removed
```

Comment:
**fields that will are renamed**
gplus_id -> social_googleplus
so_id -> social_stackoverflow

**fields that support maps & flags & profile**
ctry_filename
pg_filename
geocode
pic_url

**Soft Delete**
deleted